tests/app/integrations/comp_and_pen/test_scheduled_message_helpers.py

```diff
@@ -165,3 +167,106 @@ def test_ut_send_scheduled_comp_and_pen_sms_calls_send_notification_with_recipie
         recipient=None,
         recipient_item=recipient_item,
     )
+
+
+@pytest.mark.parametrize(
+    'amount, formatted_amount',
+    [
+        (1123.05, '1,123.05'),
+        (1000.00, '1,000.00'),
+        (10000.00, '10,000.00'),
+        (1234567.89, '1,234,567.89'),
+        (50.5, '50.50'),
+        (0.5, '0.50'),
+        (0.0, '0.00'),
```

Comment:
I tried adding  (None, '0.00') to this parameterization. It fails with the error `TypeError: unsupported format string passed to NoneType.__format__`.

`f'{item.get("paymentAmount", 0):0,.2f}'` sets the default to `0` only if the `payment amount` key does not exist. If the `paymentAmount` key has a value of None ({`paymentAmount`: `None`}), then this code will try to format `None`. 

We could add an `if/else` statement - something like `if item.get(`paymentAmount`) is None... else`. But validating the `paymentAmount` exists is handled upstream in multiple places. Adding another if/else statement would create a redundant code. 

[See doc on `.get()`](https://www.w3schools.com/python/ref_dictionary_get.asp#:~:text=the%20value%20from-,value,return%20if%20the%20specified%20key%20does%20not%20exist.%0ADefault%20value%20None,-More%20Examples)

Reply:
[I did add a test for if the `paymentAmount` key does not exists at all. ](https://github.com/department-of-veterans-affairs/notification-api/pull/1887/files#r1693702398)